superset/assets/visualizations/big_number.js

```diff
@@ -39,7 +48,7 @@ function bigNumberVis(slice, payload) {
   const dateExt = d3.extent(data, (d) => d[0]);
   const valueExt = d3.extent(data, (d) => d[1]);
 
-  const margin = 20;
+  const margin = 30;
```

Comment:
this should really be dependent on the width of the y axis label. working on a fix for that.